src/30_Advanced/Tab_Panels_with_amp-selector.html

```diff
@@ -112,6 +114,26 @@
         <div role="tab" class="tabButton" option="b">Short Container</div>
         <div role="tabpanel" class="tabContent">Hi! I'm content</div>
     </amp-selector>
+
+    <!--
+    ### Tab panels with buttons
+    This example shows a basic example with three panels, with an option to select each tab via a button.
+    -->
+    <div class="wrapper">
```

Comment:
tipp: If you remove `class=wrapper` the div won't be shown in the source code view.